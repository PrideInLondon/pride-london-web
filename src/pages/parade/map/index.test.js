import React from 'react'
import { shallow } from 'enzyme'
import ParadeMap from '.'

describe(ParadeMap.name, () => {
  it('render correctly', () => {
    process.env.GATSBY_PARADE_MAP_ID = null
    const wrapper = shallow(<ParadeMap />)
    expect(wrapper).toMatchSnapshot()
  })
})
