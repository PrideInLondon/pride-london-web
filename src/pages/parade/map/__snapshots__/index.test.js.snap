// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParadeMap render correctly 1`] = `
<Fragment>
  <map__ParadeMapContainer
    display="flex"
    flexWrap="wrap"
    mx={0}
  >
    <Gridstyles__Column
      pb={0}
      pl={
        Array [
          20,
          20,
          20,
          30,
          90,
        ]
      }
      pr={20}
      pt={
        Array [
          40,
          40,
          0,
        ]
      }
      width={
        Array [
          1,
          1,
          0.5,
          0.45,
          0.4,
        ]
      }
    >
      <map__ParadeMapContent>
        <map__Title>
          Parade map
        </map__Title>
        <map__Subtitle>
          We've created an interactive map to help you find your way around on Parade day. Use it to find the best place to watch the Parade, find your way to our different stages/areas and find important amenities like water refill stations and toilets.
        </map__Subtitle>
        <map__DownloadPDFButton
          link={true}
          primary={true}
          to="https://assets.ctfassets.net/0ho16wyr4i9n/73s6Ny4Ota2DFEt4AjbfQm/20ac5456111dbc0d177a7c0688bef835/Pride_in_London_Parade_Map_2019.pdf"
        >
          Download map as a PDF
        </map__DownloadPDFButton>
        <map__DownloadAppLabel>
          There's always poor internet connection on the day, so it's a good idea to download our app to carry the map with you offline.
        </map__DownloadAppLabel>
        <map__AppDownloadButtons>
          <map__AppDownloadButton
            aria-label="Download from App Store"
            href="https://itunes.apple.com/gb/app/pride-in-london/id1250496471"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Pride in London on the App Store"
              src=""
            />
          </map__AppDownloadButton>
          <map__AppDownloadButton
            aria-label="Download from Google Play"
            href="https://play.google.com/store/apps/details?id=org.prideinlondon.festival&hl=en"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Pride in London in the Google Play"
              src=""
            />
          </map__AppDownloadButton>
        </map__AppDownloadButtons>
        <map__DownloadPDFLinkMobile
          href="https://assets.ctfassets.net/0ho16wyr4i9n/73s6Ny4Ota2DFEt4AjbfQm/20ac5456111dbc0d177a7c0688bef835/Pride_in_London_Parade_Map_2019.pdf"
          rel="noopener noreferrer"
          target="_blank"
        >
          Download map as a PDF
        </map__DownloadPDFLinkMobile>
      </map__ParadeMapContent>
    </Gridstyles__Column>
    <Gridstyles__Column
      pb={0}
      pt={
        Array [
          0,
        ]
      }
      px={0}
      width={
        Array [
          1,
          1,
          0.5,
          0.55,
          0.6,
        ]
      }
    >
      <map__Map
        allowFullScreen={true}
        frameBorder="0"
        src="https://www.google.com/maps/d/embed?mid=null&z=16&ll=51.51004%2C%20-0.13501&hl=en&output=embed&t=m&style=feature%3Apoi%7Cvisibility%3Aoff"
      />
    </Gridstyles__Column>
  </map__ParadeMapContainer>
</Fragment>
`;
